spec/services/step_content_parser_spec.rb

```diff
@@ -47,6 +47,7 @@
 
           Conducted by Mr Andrew Preview
 
+
```

Comment:
Nitpick police: Why was this line added?

Reply:
That's because the single blank line didn't reveal the issue :-(